src/utils/parseHeadingId.ts

```diff
@@ -1,12 +1,6 @@
-const check = '{#';
+import { getKebabCaseFromName } from './';
 
-const nameToKebabCase = (name: string): string =>
-  name
-    .replace(/[#]/g, '')
-    .trim()
-    .toLowerCase()
-    .replace(/ /g, '-')
-    .replace(/[^a-z0-9-]/g, '');
+const check = '{#';
```

Comment:
any reason to define `check` as a global var here?

Reply:
Good question. That was there before I just left it.. I'm not sure it really needs to be global honestly since it's a pretty specific use case.